oauth2/oauth2-resource-server/src/main/java/org/springframework/security/oauth2/server/resource/authentication/JwtIssuerAuthenticationManagerResolver.java

```diff
@@ -130,6 +130,16 @@ public AuthenticationManager resolve(HttpServletRequest request) {
 		return authenticationManager;
 	}
 
+	/**
+	 * Set a custom issuer converter
+	 *
+	 * @since 5.5
+	 */
+	public void setIssuerConverter(Converter<HttpServletRequest, String> issuerConverter) {
```

Comment:
Instead of `setIssuerConverter`, could you do:

```java
public void setBearerTokenResolver(BearerTokenResolver bearerTokenResolver) {
    this.issuerConverter = new JwtClaimIssuerConverter(bearerTokenResolver);
}
```

That way, the API uses a familiar interface that they most likely already configured in their app.

Reply:
Okay :) Then I would also need to add another constructor to the `JwtClaimIssuerConverter` to pass the resolver / converter correct?

Reply:
@jzheaux I hope this is the way your feedback was meant.